instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/rum/RumMonitorE2ETests.kt

```diff
@@ -624,6 +625,42 @@ class RumMonitorE2ETests {
         }
     }
 
+    /**
+     * apiMethodSignature: com.datadog.android.rum.RumMonitor#fun stopResourceWithError(String, Int?, String, RumErrorSource, String, String?, Map<String, Any?> = emptyMap())
+     */
+    @Test
+    fun rum_rummonitor_stop_resource_with_error_stacktrace() {
+        val testMethodName = "rum_rummonitor_stop_resource_with_error_stacktrace"
+        val viewKey = forge.aViewKey()
+        val viewName = forge.aViewName()
+        val resourceKey = forge.aResourceKey()
+        val statusCode = forge.anInt(min = 400, max = 511)
+        val message = forge.aResourceErrorMessage()
+        val source = forge.aValueFrom(RumErrorSource::class.java)
+        val throwable = forge.aThrowable()
+        val stackTrace = throwable.loggableStackTrace()
```

Comment:
Why do you use a "real" stack trace here ? It'd be better to use `forge.aString()` to ensure that _any_ String will still work and not jus strings formatted like a java stack trace

Reply:
Was thinking to use the real case but you are right, as the stackTrace can look different when coming from other platforms better to use a random string here.